stopfmt.py

```diff
@@ -7,32 +7,31 @@
 def find_short_ifs(view, settings):
     folds = []
     cur_pt = 0
-    for i in range(1000): # limited in case bugs, could be while True
-        if_stmt = view.find(r" +if[ \(](.*)[ \)]\{\n[^\{\}\n]*\n\s*\}", cur_pt)
+
+    for _ in range(1000): # limited in case bugs, could be while True
+        if_stmt = view.find(r"\s*if[ \(](.*)[ \)]\{\n[^\{\}\n]*\n\s*\}", cur_pt)
         if if_stmt is None or if_stmt.empty():
             break # no more found
 
+        # move past this one
+        cur_pt = if_stmt.end()
 
         # check that if we folded it the line wouldn't be too long
         stmt = view.substr(if_stmt)
         stmt = REMOVE_WHITESPACE.sub(" ", stmt)
         if len(stmt) > settings.get('max_line_length', 100):
             continue
 
-        # print(stmt)
-
         # find the parts to fold, this is kinda overkill
         sub_pt = if_stmt.begin()
         for i in range(10): # limited in case bugs, could be while True
             line_break = view.find(r"\n\s*", sub_pt)
             if line_break is None or line_break.empty() or line_break.begin() >= if_stmt.end():
                 break # no more found
-            folds.append(line_break)
+            if (i > 0):
```

Comment:
in case of 
```
}
if x {
```

Reply:
Huh I wonder why this didn't go wrong for me, I think the better solution than this would be to change the if statement searching regex from starting with `\s*` to starting with `[ \t]*`

Reply:
not sure, I'll update the regex again because we also missed `if (){ ...} else{}`